views/campaign-preview.twig

```diff
@@ -13,19 +13,19 @@ This is the template for the campaign page
     <div class="widget">
         <div class="widget-title">{% trans "Campaign preview" %}</div>
         <div class="widget-body">
-            <p><b>total duration</b> {{ duration }} <i>(hours:min:sec)</i></p>
-            <p><b>number of layouts</b> : {{ campaign.numberLayouts }} </p>
+            <p><b>{% trans "total duration" %}</b> {{ duration }} <i>({% trans "hours:min:sec" %})</i></p>
+            <p><b>{% trans "number of layouts" %}</b> : {{ campaign.numberLayouts }} </p>
             <div class="row clearfix">
-              {% for layout in layouts %}
+              {% for extendedLayout in extendedLayouts %}
                 <div class="col-sm-6 col-md-4 col-lg-3">
                   <div class="embed-responsive embed-responsive-4by3">
-                      <div class="embed-responsive-item preview-container">
-                        {% include 'layout-preview-partial.twig' with {'layout': layout, 'previewOptions': previewOptions } %}
-                      </div>
+                        <div class="embed-responsive-item preview-container" data-url="/layout/preview/{{ extendedLayout.layout.layoutId }}">
```

Comment:
Should use `{{ urlFor("<<theroutename>>", { "id": extendedLayout.layout.layoutId }) }}` so that the path is always resolved correctly (people might be running in virtual folders, sub folders, etc)